packages/sha/package.js

```diff
@@ -1,9 +1,12 @@
 Package.describe({
-  summary: "SHA256 implementation",
-  version: "1.0.4-plugins.0"
+  name: 'sha256',
+  version: '1.0.4-plugins.0'
+  summary: 'SHA256 implementation',
+  git: 'https://github.com/meteor/meteor/tree/devel/packages/sha',
+  documentation: 'README.md'
 });
```

Comment:
Can we please have the `git` field here so https://atmospherejs.com/meteor/sha has a GitHub/issues link?